src/sentry/models/project.py

```diff
@@ -104,11 +104,12 @@ class Project(Model, PendingDeletionMixin):
     first_event = models.DateTimeField(null=True)
     flags = BitField(
         flags=(
+            ("has_alert_filters", "This Project has filters"),
             ("has_releases", "This Project has sent release data"),
             ("has_issue_alerts_targeting", "This Project has issue alerts targeting"),
             ("has_transactions", "This Project has sent transactions"),
         ),
-        default=2,
+        default=10,
```

Comment:
not sure if I did this correctly, but I tried to maintain the default for `has_issue_alerts_targeting` while also making projects have the `has_alert_filters` flag turned on

Reply:
actually I'm not sure if setting the default here is what I want. Ideally what I'd want to have happen is that future projects receive `has_alert_filters` to be turned on, but existing projects have it turned off.